lib/Negative.js

```diff
@@ -1,55 +1,31 @@
-const Node = require('./node');
+const clone = require('./util/clone');
+const {build, query} = require('math-nodes');
 
 const Negative = {};
 
-// Returns if the given node is negative. Treats a unary minus as a negative,
```

Comment:
IsNegative is now a helper function in `math-nodes`. https://github.com/semantic-math/math-nodes/blob/master/lib/query.js#L23